File: API/Models/Rating/RecentRating.cs
namespace Models.Rating
{
  public class RecentRating
  {
    public string? AlbumTitle { get; set; }
    public string? AlbumImageURL { get; set; }
    public string? AlbumId { get; set; }
    public string? ArtistName { get; set; }
    public string? ArtistId { get; set; }
    public DateTime DateOfRating { get; set; }
    public uint Grade { get; set; }
  }
}

File: Client/js/Ratings/RecentRating.js
export class RecentRating{
  constructor(recentRating)
  {
    this.albumTitle = recentRating.albumTitle;
    this.albumImageUrl = recentRating.albumImageURL;
    this.albumId = recentRating.albumId;
    this.artistName = recentRating.artistName;
    this.dateOfRating = recentRating.dateOfRating;
    this.artistId = recentRating.artistId;
    this.grade = recentRating.grade;
  }
}